Add `Senal-D-A-Gunaratna/hyprfade.nvim` (#2437)
rockerBOO/awesome-neovim: README.md

- [s1n7ax/nvim-terminal](https://github.com/s1n7ax/nvim-terminal) - A simple and easy to use multi-terminal plugin.
- [logicmagix/tide42](https://github.com/logicmagix/tide42) - A fully integrated terminal IDE built on Lua, tmux, and scriptable workflows.
- [samjwill/nvim-unception](https://github.com/samjwill/nvim-unception) - Automatic unnesting of editor sessions started from terminal buffers.

> - [Senal-D-A-Gunaratna/hyprfade.nvim](https://github.com/Senal-D-A-Gunaratna/hyprfade.nvim) - Fade the terminal window hosting Neovim on Hyprland via `hyprctl` window opacity.

Reviewer: ```suggestion
- [Senal-D-A-Gunaratna/hyprfade.nvim](https://github.com/Senal-D-A-Gunaratna/hyprfade.nvim) - When using Hyprland, fade the terminal window via `hyprctl`.
```

- [iamironz/android-nvim-plugin](https://github.com/iamironz/android-nvim-plugin) - Android build, deploy, and logcat commands with Gradle integration.
- [massix/termux.nvim](https://github.com/massix/termux.nvim) - Interact with Termux APIs, useful to gather various information about your Android phone to display in the statusline (e.g. battery level).
- [m15a/flake-awesome-neovim-plugins](https://github.com/m15a/flake-awesome-neovim-plugins) - Nix flake that provides a collection of `awesome-neovim` plugins as Nix packages.

> - [Senal-D-A-Gunaratna/hyprfade.nvim](https://github.com/Senal-D-A-Gunaratna/hyprfade.nvim) - Fade the terminal window hosting Neovim on Hyprland via `hyprctl` window opacity.

Reviewer: I think this fits better in the UI section. @Penaz91 what are your thoughts?

Reviewer 2: Personally I intend "UI" for something that enhances/changes Neovim's UI. So I'm not sure it's the best section.

"Terminal Integration" or similar might be the way?

Author: @Penaz91  should i relocate it?

Reviewer 2: Sorry for the delay, I would like to know DrKJeff's opinion on the matter.

Thanks for your patience, for now let's leave it as-is. I'll make sure to mention you if anything is needed!

Reviewer: Yes, Terminal Integration is appropriate.

Reviewer 2: @Senal-D-A-Gunaratna Thanks for waiting. Please move the plugin under the "Terminal Integration" category.
